store/works.js

```diff
@@ -1,9 +1,31 @@
+import { action } from 'mobx'
+
 import { Pages } from './helpers/pages'
+import apiRequest from '../api'
 
 class Works extends Pages {
   constructor(baseUrl) {
     const url = `${baseUrl}/works`
     super(url)
+    this.worksUrl = url
+  }
+
+  @action
+  async changeVisibility(rowId, isVisible) {
+    const [pageNumber, workId] = rowId.split('-', 2)
+    await apiRequest(
+      `${this.worksUrl}/${workId}`,
+      'PUT',
```

Comment:
It should be PATCH because the update is partial. You can find the [classification on Wikipedia](https://en.wikipedia.org/wiki/Representational_state_transfer#Relationship_between_URI_and_HTTP_methods).

Reply:
Well, I used what was implemented. So if they change it on the backend I don't mind it to change here.

Reply:
@valerapro is aware of it